bundles/org.openhab.binding.groheondus/pom.xml

```diff
@@ -15,14 +15,14 @@
   <name>openHAB Add-ons :: Bundles :: GROHE ONDUS Binding</name>
 
   <properties>
-    <dep.noembedding>commons-text,commons-lang3</dep.noembedding>
+    <dep.noembedding>commons-lang3</dep.noembedding>
```

Comment:
Nice, this happened to address https://github.com/openhab/openhab-addons/security/dependabot/29